renv/profiles/dev/renv.lock

```diff
@@ -469,7 +514,179 @@
         "utils",
         "zip"
       ],
-      "Hash": "c03b4c18d42da881fb8e15a085c2b9d6"
+      "Hash": "14304e44a0f90fa2d0f905472333c561"
+    },
+    "paws": {
```

Comment:
This SDK is technically a `noctua` dependency. I'm not sure why they were missing before.